app/uk/gov/hmrc/gform/submission/DestinationAuditController.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.submission
+
+import play.api.Logger
+import play.api.libs.json.Json
+import play.api.mvc.{ Action, AnyContent }
+import uk.gov.hmrc.gform.controllers.BaseController
+
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Do not use `Implicits.global`. This is wrong as it doesn't propagate diagnostic information.

All usages in gform code base are wrong and needs to be fixed (except in `test`).

Reply:
Yes we should use the MDCLoggingExecutionContext